### PR TITLE
Use ReactDom instead React to get rid of React warnings

### PR DESCRIPTION
**Pull request by valichek**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,17 +15,17 @@
...
     // that support module.exports
-    module.exports = factory(require('react'));
+    module.exports = factory(require('react-dom'));
   } else {
     // Browser globals (root is window)
-    root.OnClickOutside = factory(React);
+    root.OnClickOutside = factory(ReactDom);
   }
-}(this, function (React) {
+}(this, function (ReactDom) {
```
Comment (Pomax) [inline index.js]: let's keep this `React` though

File: index.js [hunk 2]
```diff
@@ -72,7 +72,7 @@
           }
           eventHandler(evt);
         }
-      }(React.findDOMNode(this), this.handleClickOutside));
+      }(ReactDom.findDOMNode(this), this.handleClickOutside));
```
Comment (Pomax) [inline index.js]: and the same here

File: index.js (hunk 1 above)
Comment (eliseumds) [inline index.js]: The method *findDOMNode* is available at react-dom.

**Comment by eliseumds** [comment]
Guys, this is a big update, isn't it? ReactDOM is released with React 0.14, so, besides increasing the major version, react >= 0.14 should be specified in *package.json*.

**Comment by Pomax** [comment]
this hasn't landed. If it does, then this would be a major update, yes.

**Comment by Pomax** [comment]
landed as major update, on npm as v4.0.1

**Pull request closed without merging**